Sources/DriverSupport/DriverSupportUtils.swift

```diff
@@ -43,21 +56,30 @@ public class DriverSupport {
     // This checks if given flags are supported in the built-in toolchain driver. Currently
     // there's no good way to get the supported flags from it, so run `swiftc -h` directly
     // to get the flags and cache the result.
-    public func checkToolchainDriverFlags(flags: Set<String>, toolchain: PackageModel.Toolchain, fileSystem: FileSystem) -> Bool {
-        let trimmedFlagSet = Set(flags.map{$0.trimmingCharacters(in: ["-"])})
-        if let entry = flagsMap.get(), let cachedSupportedFlagSet = entry[toolchain.swiftCompilerPath.pathString + "-driver"] {
+    public func checkToolchainDriverFlags(
+        flags: Set<String>,
+        toolchain: PackageModel.Toolchain,
+        fileSystem: FileSystem
+    ) -> Bool {
+        let trimmedFlagSet = Set(flags.map { $0.trimmingCharacters(in: ["-"]) })
+        let swiftcPathString = toolchain.swiftCompilerPath.pathString
+
+        if let entry = flagsMap.get(), let cachedSupportedFlagSet = entry[swiftcPathString + "-driver"] {
             return cachedSupportedFlagSet.intersection(trimmedFlagSet) == trimmedFlagSet
         }
         do {
-            let helpJob = try Process.launchProcess(arguments: ["swiftc", "-h"], env: ProcessEnv.vars)
+            let helpJob = try Process.launchProcess(
+                arguments: [swiftcPathString, "-h"],
```

Comment:
It's a bit hard to identify the actual change here, since it seems mostly doing formatting, but I think this is the only thing that changes?

Reply:
Yes, correct. I've also moved factored out `pathString` property access to a separate `let` binding for reuse in multiple places. Sorry about that, I'll try to keep formatting changes to separate PRs then.